fibonacci-series.php

```diff
@@ -0,0 +1,26 @@
+<?php 
+// PHP program to print first n Fibonacci numbers 
+  
+// Function to print first n Fibonacci Numbers 
+
+function printFibonacciNumbers($n) 
+{ 
+    $f1 = 0; 
+    $f2 = 1;  
+    $i; 
+  
+    if ($n < 1) 
+        return; 
+  
+    for ($i = 1; $i <= $n; $i++) 
+    { 
+        echo($f2); 
+        echo(" "); 
```

Comment:
Codacy found an issue: [Use of echo language construct is discouraged.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6328068)